test/installation/node/install from source without V8 snapshot for ARM

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+die () { echo $@ ; exit 1; }
+
+. ../../../nvm.sh
+
+NVM_TEST_VERSION=v0.10.7
+
+# Remove the stuff we're clobbering.
+[ -e ../../../$NVM_TEST_VERSION ] && rm -R ../../../$NVM_TEST_VERSION
+
+# Fake ARM arch
+nvm_get_arch() {
+  echo "armv7l"
+}
+
+# Install from source
+nvm install -s $NVM_TEST_VERSION || die "'nvm install -s $NVM_TEST_VERSION' failed"
+
+# Check Install
+[ -d ../../../$NVM_TEST_VERSION ]
+nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION || "'nvm run $NVM_TEST_VERSION --version | grep $NVM_TEST_VERSION' failed"
+
+# Check V8 snapshot isn't compiled
+nvm run $NVM_TEST_VERSION -p process.config | grep "v8_use_snapshot: false" || "'nvm run $NVM_TEST_VERSION -p process.config | grep \"v8_use_snapshot: false\"' failed"
```

Comment:
I note that when I run this command on my own system, it says `v8_use_snapshot: 1` - is there a chance it will sometimes be 0/1, and sometimes true/false?